src/inputs/labeled-inputs.scss

```diff
@@ -55,6 +55,10 @@
     // stylelint-enable
   }
 
+  .iui-input-group .iui-toggle-switch-wrapper {
+    padding: round($iui-baseline * 0.5) 0;
```

Comment:
What is this for?

Reply:
#315 wants the padding removed from the toggle switch, so I placed this within `labeled-inputs.scss` to avoid causing issues.